jobserver/templates/project_detail.html

```diff
@@ -32,29 +33,86 @@ <h2>{{ project.name }}</h2>
       {% endif %}
     </div>
 
-    <div class="d-flex justify-content-between align-items-center">
-      <h3>Workspaces</h3>
-
-      {% if can_manage_workspaces %}
-      <a class="btn btn-sm btn-primary" href="{% url 'workspace-create' org_slug=project.org.slug project_slug=project.slug %}">
-        Create a Workspace
-      </a>
+    <div class="mb-4">
+      {% if project.description %}
+      <p>{{ project.description }}
       {% endif %}
+
+      <p>
+        This is an OpenSAFELY project. Every time a researcher runs their
+        analytic code against patient data, it is audited in public here. Use
```

Comment:
We should either say "collaborator" here or "researchers" in the heading above the list of users.

I think the latter...